User-Data Store Migration/README.md

```diff
@@ -0,0 +1,85 @@
+# User-Data Store Migration Tool
+
+## Context
+Linkurious Enterprise is shipped with an embedded Sqlite database
+to allow an immediate usage of the system in a PoC environment, however it's a best practice to relay on a relational databased management system (RDBMS) in a production environment.
+
+It's a common need of our customers to migrate the data from the Sqlite database to an external relational database.
+
+Since different databases may have different peculiarities, this tools is aimed to provide the most reliable process to safely migrate the data.
+
+> /!\ The tool's objective is to migrate data from Sqlite to an external db for the same Linkurious Enterprise instance. No other scenario are covered.
+
+# The migration process
+
+Linkurious Enterprise is designed to create and maintain (through the updates) it's data structure over the supported databases.
+
+The safest migration approach is to let the system handle the whole task of defining the db schema and then perform manual tasks only to reimport the data.
+
+The tool is designed to help a database administrator to generate the correct script to import data staring from a dump. In particular the script will:
```

Comment:
I advise you to split: "how to use the tool" and "what the tool does".
The former should appear first, as it is more important to users.

Reply:
I don't fully agree on this...before using the tool I guess it's important to understand what and why the tool is designed for.

As soon as this is clear, all the instructions are more clear. e.g. `The tool is designed to help a database administrator to generate` I'm not expecting anyone that don't know anything of db to use the tool. WDYT?

However I can move the `The tool is designed [...]` section in `# How to use / customize the tool`. Is it ok?

Reply:
Yes, it makes sense to explain who the tools is targeted for in the "how to use" section.
Regarding splitting into two sections, it's up to you, this was simply a piece of advice.

Reply:
d566f17